cli/lib/appdata.ts

```diff
@@ -92,9 +102,24 @@ export async function saveAppdata( userData: UserData ): Promise< void > {
 		const fileContent = JSON.stringify( userData, null, 2 ) + '\n';
 
 		fs.writeFileSync( appDataPath, fileContent, 'utf8' );
+	} catch ( error ) {
+		throw new LoggerError( __( 'Failed to save appdata file' ), error );
+	}
+}
+
+export async function getAuthToken(): Promise< NonNullable< UserData[ 'authToken' ] > > {
```

Comment:
This function had already shrunken noticeably compared to the original implementation, so I figured moving it to `appdata.ts` was a better fit.